@navikt/core/react/src/form/combobox/Input/Input.context.tsx

```diff
@@ -47,8 +61,6 @@ const InputProvider = ({ children, value: props }) => {
   } = props;
   const formFieldProps = useFormField(
     {
-      description,
```

Comment:
I see that we use `description` straight from the props, which I suppose is why you are removing it from this file. But it is also used to create `inputDescriptionId` in `useFormField`, which we do use. Without this, we won't connect the input field with the description.

I think we should rather change Combobox.tsx to get description from useInputContext (which would use formFieldProps created here).

Reply:
ofc, the real problem is that `description` and `disabled` wasn't being sent into `InputContextProvider`.

I have fixed `description` now, but what should we do with `disabled`? Today, setting disabled doesn't do anything. I could send it into `useFormField` as well, but that would only disable the input field itself, not the buttons etc. Is that ok for now, then we can fix it properly in a different branch? I feel it's outside of the scope of this PR 😅

Reply:
I think that is a good solution for now. We should probably fix it properly, but I agree that is scope creep for this PR.